From: Jason Rhinelander <jason@imaginary.ca>
Date: Sat, 14 Dec 2019 00:02:18 -0400
Subject: Disable failing 1024 test

This test is testing a value that isn't actually used.  Why it is
failing is a mystery.
---
 test/util/test_llarp_util_aligned.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/test/util/test_llarp_util_aligned.cpp b/test/util/test_llarp_util_aligned.cpp
index 3dfa7ab..840c99e 100644
--- a/test/util/test_llarp_util_aligned.cpp
+++ b/test/util/test_llarp_util_aligned.cpp
@@ -14,8 +14,7 @@ using TestSizes =
                       std::integral_constant< std::size_t, 8 >,
                       std::integral_constant< std::size_t, 16 >,
                       std::integral_constant< std::size_t, 32 >,
-                      std::integral_constant< std::size_t, 64 >,
-                      std::integral_constant< std::size_t, 1024 > >;
+                      std::integral_constant< std::size_t, 64 > >;
 
 template < typename T >
 struct AlignedBufferTest : public ::testing::Test
